modules/ROOT/pages/apikit-odatav4-extension-reference.adoc

```diff
@@ -81,48 +76,46 @@ Default configuration
 
 === For Configurations
 
-* <<Config>> {nbsp}
+* <<Config>> 
 
 === Throws
 
-* APIKIT-ODATA:DESERIALIZATION {nbsp}
-* APIKIT-ODATA:INTERNAL_SERVER_ERROR {nbsp}
-* APIKIT-ODATA:MALFORMED_SEARCH {nbsp}
-* APIKIT-ODATA:MALFORMED_URI {nbsp}
-* APIKIT-ODATA:NOT_FOUND {nbsp}
-* APIKIT-ODATA:NOT_IMPLEMENTED {nbsp}
-* APIKIT-ODATA:SERIALIZATION {nbsp}
-* APIKIT-ODATA:TRANSFORMATION {nbsp}
-
+* APIKIT-ODATA:DESERIALIZATION 
+* APIKIT-ODATA:INTERNAL_SERVER_ERROR 
+* APIKIT-ODATA:MALFORMED_SEARCH 
+* APIKIT-ODATA:MALFORMED_URI 
+* APIKIT-ODATA:NOT_FOUND 
+* APIKIT-ODATA:NOT_IMPLEMENTED 
+* APIKIT-ODATA:SERIALIZATION 
+* APIKIT-ODATA:TRANSFORMATION 
```

Comment:
```suggestion
* APIKIT-ODATA:TRANSFORMATION 
* APIKIT-ODATA:EXPANSION
```

I'm really really sorry for this dumb mistake. The new error goes here, not on the source (as said on the documentation: you can't catch expansion errors on the source where they happen).

Reply:
No problem!
Thanks for fixing it!